packages/components/src/box-control/utils.ts

```diff
@@ -199,13 +204,17 @@ export function normalizeSides( sides ) {
  * Applies a value to an object representing top, right, bottom and left sides
  * while taking into account any custom side configuration.
  *
- * @param {Object}        currentValues The current values for each side.
- * @param {string|number} newValue      The value to apply to the sides object.
- * @param {string[]}      sides         Array defining valid sides.
+ * @param currentValues The current values for each side.
+ * @param newValue      The value to apply to the sides object.
+ * @param sides         Array defining valid sides.
  *
- * @return {Object} Object containing the updated values for each side.
+ * @return Object containing the updated values for each side.
  */
-export function applyValueToSides( currentValues, newValue, sides ) {
+export function applyValueToSides(
+	currentValues: BoxControlValue,
+	newValue: string | undefined,
```

Comment:
Nit – any reason for not using the optional syntax here?

```suggestion
	newValue?: string,
```

Reply:
Hmm, at some point it complained about `?` syntax on params preceding another optional param, but it's not happening anymore 🤷🤷🤷 Reverted to normal 1b1d1e5c